apis/metal3.io/v1alpha1/baremetalhost_types.go

```diff
@@ -123,6 +123,9 @@ const (
 type ErrorType string
 
 const (
+	// AdoptionError is an error condition occurring when the controller
+	// is unable to adopt an already provisioned host.
+	AdoptionError ErrorType = "adoption error"
```

Comment:
Need to make this an allowed value on line 525 also, otherwise we won't be able to write the status.

Reply:
Thanks! That was the trick - it shows up in the status field now